packages/expo-cli/src/commands/info/doctor/doctorAsync.ts

```diff
@@ -32,7 +32,23 @@ async function validateSupportPackagesAsync(sdkVersion: string): Promise<boolean
       }
     }
   }
+
+  // None of these packages should exist in any modern project
+  const illegalPackages = [
+    '@unimodules/core',
+    '@unimodules/react-native-adapter',
+    'react-native-unimodules',
+  ];
+
+  for (const pkg of illegalPackages) {
+    const isPackageAbsent = await warnAboutDeepDependenciesAsync({ name: pkg });
+    if (!isPackageAbsent) {
+      allPackagesValid = false;
+    }
```

Comment:
We want them to be absent, right? So how does the fact that a package is absent toggle `allPackagesValid` to `false`? This is probably a naming issue, and not a logic issue.

Reply:
i clarified in naming in latest commit